File: 1672_leet/ans.c
int maximumWealth(int** accounts, int accountsSize, int* accountsColSize){
    int max = 0;
    int i, j, sum;

    for (i = 0; i < accountsSize; i++)
    {
        sum = 0;
        for (j = 0; j < accountsColSize[i]; j++)
        {
            sum += accounts[i][j];
        }
        if (sum > max)
            max = sum;
    }
    return (max);
}

File: 1672_leet/ans.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        # return max([reduce((lambda x, y: x + y), account) for account in accounts])
        # return max(map(sum, accounts))
        return max([sum(x) for x in accounts])

# class Solution:
#     def maximumWealth(self, accounts: List[List[int]]) -> int:
#         max = 0
#         for acc in accounts:
#             sum = 0
#             for i in acc:
#                 sum += i
#             else:
#                 max = sum if sum > max else max
#         return max